sons-of-the-forestmetaconfig.json

```diff
@@ -0,0 +1,7 @@
+[
+    {
+        "ConfigFile": "saves/dedicatedserver.cfg",
```

Comment:
Shouldn't it be:
`sons-of-the-forestdedicatedserver.cfg`?

Reply:
Nope! The update stage renames it to what the server needs. :)

Reply:
That's damn confusing. Great work!